src/DynamoCore/Graph/Workspaces/HomeWorkspaceModel.cs

```diff
@@ -331,9 +332,16 @@ internal override void RequestRun()
         {
             base.RequestRun();
 
-            if (RunSettings.RunEnabled && RunSettings.RunType != RunType.Manual)
+            if (RunSettings.RunType != RunType.Manual)
             {
-                Run();
+                // TODO for Dynamo 3.0: The boolean "executingTask" that is used here is a make-do fix.
+                // We will be needing a separate variable(boolean) to check for RunEnabled flag from external applications and
```

Comment:
This can be worded slightly differently, maybe like this: `We will be needing a separate variable (boolean flag) to check whether run can be enabled from external applications and ...`